src/Recruit/SFA.DAS.Recruit.sln

```diff
@@ -42,9 +42,8 @@ Global
 		{D5D4AB58-5E91-42DF-ABFF-E03BCDCD83DB}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{D5D4AB58-5E91-42DF-ABFF-E03BCDCD83DB}.Release|Any CPU.Build.0 = Release|Any CPU
 		{41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-        {41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Debug|Any CPU.Build.0 = Debug|Any CPU
-        {41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Release|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
Is this change required?

Reply:
no, I'll revert.  VS did that, not me...